build.rs

```diff
@@ -2,6 +2,38 @@ extern crate ructe;
 extern crate rsass;
 use ructe::*;
 use std::{env, fs::*, io::Write, path::PathBuf};
+use std::process::{Command, Stdio};
+
+fn compute_static_hash() -> String {
+    //"find static/ -type f ! -path 'static/media/*' | sort | xargs stat --printf='%n %Y\n' | sha256sum"
+
+    let find = Command::new("find")
+        .args(&["static/", "-type", "f", "!", "-path", "static/media/*"])
+        .stdout(Stdio::piped())
+        .spawn()
+        .expect("failed find command");
+
+    let sort = Command::new("sort")
```

Comment:
btw, is there no way to do this in rust?

Reply:
I think there are plenty, but I'm lazy and it's a build script. I prototyped using bash, so my rust code is basically bash translated (if you find it really bad, I can change that but I don't know if it's really useful)

Reply:
nah, i was just curious